code/modules/mob/living/living.dm

```diff
@@ -535,9 +535,15 @@
 		clear_fullscreen("tint", 0)
 		return FALSE
 
+///Modifies the mobs inherent accuracy modifier
 /mob/living/proc/adjust_mob_accuracy(accuracy_mod)
 	ranged_accuracy_mod += accuracy_mod
+	SEND_SIGNAL(src, COMSIG_RANGED_ACCURACY_MOD_CHANGED, ranged_accuracy_mod)
 
+///Modifies the mobs inherent scatter modifier
+/mob/living/proc/adjust_mob_scatter(scatter_mod)
+	ranged_scatter_mod += scatter_mod
+	SEND_SIGNAL(src, COMSIG_RANGED_SCATTER_MOD_CHANGED, ranged_scatter_mod)
```

Comment:
pass amount changed not the var people can fetch that off the source ref

Reply:
done.